apps/glusterfs/node_entry_test.go

```diff
@@ -22,11 +22,12 @@ import (
 )
 
 func createSampleNodeEntry() *NodeEntry {
+	g := utils.GenUUID()
 	req := &api.NodeAddRequest{
 		ClusterId: "123",
 		Hostnames: api.HostAddresses{
-			Manage:  []string{"manage" + utils.GenUUID()[:8]},
-			Storage: []string{"storage" + utils.GenUUID()[:8]},
+			Manage:  []string{"manage" + g[:4] + g[24:]},
+			Storage: []string{"storage" + g[:4] + g[24:]},
```

Comment:
This also changes the code to use the same uuid (pieces) instead of generating a new one, for manage and storage. Intended?

Reply:
I think it should be fine because its just generating fake "hosts" for the db entires for the test. They only need to be across nodes not within the the node. 

That said while I was commuting home yesterday I was thinking that these changes were a bit ugly. What do you think about adding a "short id" generating function that is meant specifically for cases like this within the test code? Something like:

```golang
// ShortId returns a truncated id that is unique-ish but short and convenient for use
// within test code. Specify the length of this id as the first argument. If this length is
// not valid the function will panic.
func ShortId(s int) string {
  // ... stuff ...
}
```

Reply:
As a  side note, I would love to be able to use shortened UUIDs on the heketi-cli and have them match the actual longer ones. Just like git commands handle the hashes... But, that is probably a db feature, and not related to the UUID generation :-)